benchmarks/src/bench.rs

```diff
@@ -57,6 +62,38 @@ pub static TARGET_ITERATION_COUNT: Lazy<usize> = Lazy::new(|| {
         .expect("invalid TARGET_ITERATION_COUNT")
 });
 
+static SMALL_DOC: OnceCell<Document> = OnceCell::const_new();
```

Comment:
Nit: I think these statics can be moved into the scope of the `get_` functions that use them.